File: src/main/java/no/nav/service/pensjon/regler/to/BeregnOpptjeningRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;

import no.nav.domain.pensjon.regler.grunnlag.Pensjonsbeholdning;
import no.nav.domain.pensjon.regler.grunnlag.Persongrunnlag;
/**
 * Dataoverføringsobjekt for tjenesten beregnOpptjening.
 */
public class BeregnOpptjeningRequest extends ServiceRequest {
	private Date beholdningTom;
	private Persongrunnlag persongrunnlag;
	private Pensjonsbeholdning beholdning;


}

File: src/main/java/no/nav/pensjon/regler/to/BeregnOpptjeningRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.grunnlag.Pensjonsbeholdning
import no.nav.pensjon.regler.domain.grunnlag.Persongrunnlag
import java.util.*

/**
 * Dataoverføringsobjekt for tjenesten beregnOpptjening.
 */
class BeregnOpptjeningRequest {
    var beholdningTom: Date? = null
    var persongrunnlag: Persongrunnlag? = null
    var beholdning: Pensjonsbeholdning? = null
}